Topic 1/src/com/globant/topic1/exercise7/model/PlayList.java

```diff
@@ -0,0 +1,56 @@
+package com.globant.topic1.exercise7.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlayList {
+
+	private List<Song> songs;
+
+	public PlayList() {
+		this.songs = new ArrayList<>();
+	}
+
+	public PlayList(List<Song> songs) {
+		this.songs = songs;
+	}
+
+	public List<Song> getSongs() {
+		return songs;
+	}
+
+	public void setSongs(List<Song> songs) {
+		this.songs = songs;
+	}
+
+	public void addSong(Song song) {
+		songs.add(song);
+		System.out.println(song + " added to the playlist");
+	}
+
+	public void removeSong(Song song) {
+		if (songs.isEmpty()) {
+			System.out.println("The playlist is empty");
+		} else if (songs.contains(song)) {
+			System.out.println("The song " + song + " was removed");
```

Comment:
we can actually do the remove here. 
check this out: https://docs.oracle.com/javase/7/docs/api/java/util/ArrayList.html#remove(java.lang.Object)